composer.json

```diff
@@ -3,39 +3,53 @@
 	"description": "Ecommerce module to create online shops using SilverStripe.",
 	"homepage": "http://swipestripe.com",
 	"license": "BSD-3-Clause",
-	"type": "silverstripe-module",
+	"type": "silverstripe-vendormodule",
 	"keywords": ["silverstripe", "ecommerce", "e-commerce", "shop", "swipestripe", "swipe stripe"],
 	"authors": [
 		{
-			"name": "Frank Mullenger",
-			"email": "frankmullenger@gmail.com"
+			"name": "Harsh Chokshi",
+			"email": "harsh.chokshi@littlegiant.co.nz"
 		}
 	],
-	"require":
-	{
-		"silverstripe/framework": "~3.1",
-		"silverstripe/cms": "~3.1",
-		"silverstripe/payment": "1.0.*"
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"require": {
+		"php": ">=7.1",
+		"ext-intl": "*",
+		"silverstripe/framework": "^4",
+		"silverstripe/versioned": "^1.2",
+		"silverstripe/cms": "^4.2",
+        "silverstripe/silverstripe-omnipay": "dev-feature/omnipay-v3",
+        "hchokshi/silverstripe-trait-config": "^1.0"
```

Comment:
I don't feel hugely strongly about this, but as a general rule we shouldn't be adding our personal projects as dependencies to open source modules. We want this to be a community run project. Shifting such a module to the same space (e.g. swipestripe/trait-config) and making this an optional / suggested addon would be better.

I suggest switching this to littlegiant organisation for now, and re-vendoring there.

For my personal projects I've started shifting stuff into an organisation and adding collaborators onto that.

Reply:
Removed in a2b843f due to issues loading `TraitConfigTransformer` into `TestKernel`.